src/lib/features/feature-links/feature-links-read-model-type.ts

```diff
@@ -2,9 +2,10 @@ export interface IFeatureLink {
     id: string;
     url: string;
     title: string | null;
+    feature: string;
 }
 
 export interface IFeatureLinksReadModel {
-    getLinks(feature: string): Promise<IFeatureLink[]>;
+    getLinks(...features: string[]): Promise<IFeatureLink[]>;
```

Comment:
making this method generic so that it can serve a single feature view and feature export